docs/book/element/phone-number.md

```diff
@@ -0,0 +1,94 @@
+# PhoneNumber
+
+`Laminas\I18n\PhoneNumber\Form\Element\PhoneNumber` is a text input form element that automatically attaches a PhoneNumber validator.
+
+## Basic Usage
+
+This element automatically adds a `type` attribute of value `tel`.
+
+```php
+use Laminas\I18n\PhoneNumber\Form\Element\PhoneNumber;
+use Laminas\Form\Form;
+
+$phone = new PhoneNumber('number');
+$phone->setLabel('Phone Number');
+$phone->setAttributes([
+    'autocomplete' => 'tel', // 'tel-national' is also an option.
+]);
```

Comment:
Is the autocomplete option recommended? Otherwise please add this part to the callout where the autocomplete attribute is described.